src/mame/drivers/hh_sm510.cpp

```diff
@@ -6557,6 +6641,16 @@ ROM_START( exospace )
 	ROM_LOAD( "exospace.svg", 0, 66790, BAD_DUMP CRC(df31043a) SHA1(2d8caf42894df699e469652e5f448beaebbcc1ae) )
 ROM_END
 
+ROM_START( gnw_dkong )
+        ROM_REGION( 0x1000, "maincpu", 0 )
+        ROM_LOAD( "dk-52", 0x0000, 0x1000, CRC(5180cbf8) SHA1(5174570a8d6a601226f51e972bac6735535fe11d) )
+
+        ROM_REGION( 267443, "svg_top", 0)
```

Comment:
Could you make region length same as file length? For the other screen too

Reply:
Same goes for this one, I missed that the filesize was specified two times per file. Thanks for finding it!